### PR TITLE
Add AI hologram fluff

### PR DESCRIPTION
**Pull request by Allfd**
Extends custom_sprites.txt to enable custom holograms for the AI by
adding a third entry “Hologram”.

:cl:Allfd
rscadd: Added hologram fluff sprites
/:cl:

### CODE REVIEW

**Comment by Ar3nn** [comment]
So a hologram sprite only one ckey could use?

**Comment by Allfd** [comment]
@Ar3nn 

Yep,  it extends the custom_sprites feature that has been in use for some time for custom AI and Borg sprites as donor fluff.

**Comment by Allfd** [comment]
Yep, go me and buttons.

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 1]
```diff
@@ -792,6 +794,21 @@ var/list/ai_verbs_default = list(
...
+		var/lines = splittext(file, "\n")
+
+		for(var/line in lines)
+		// split & clean up
+			var/list/Entry = splittext(line, ":")
+			for(var/i = 1 to Entry.len)
+				Entry[i] = trim(Entry[i])
+
+			if(Entry.len < 3)
+				continue;
```
Comment (Fox-McCloud) [inline code/modules/mob/living/silicon/ai/ai.dm]: no `;`

**Pull request merged**